snap/snapcraft.yaml

```diff
@@ -0,0 +1,106 @@
+name: swipl
+base: core18
+adopt-info: swi-prolog
+license: 'BSD-2-Clause'
+summary: ISO/Edinburgh-style Prolog interpreter
```

Comment:
Could you change _interpreter_ that to _environment_ or _implementation_, _system_, ...?

Reply:
Sure, I just copied this from the debian package in apt...

Reply:
Should be changed there too.  I don't really like _interpreter_.  Seems a term from the past century.   SWI-Prolog (and virtually all Prolog's) depend on a virtual machine.

Reply:
I agree. I personally use implementation, but system seems to be more future proof looking at all the non standard or simply new additions